### ISSUE TITLE
Split package warnings in gradle builds

### ISSUE THREAD

**Issue by joeha480** [brailleapps/issues#24]
This warning is caused by bnd because it detects the same packages on the class path more than once. This in turn, is because the source sets in build.gradle point to the same folder twice. This in turn, is because I have nothing but contempt for the standard directory layout (https://maven.apache.org/guides/introduction/introduction-to-the-standard-directory-layout.html). One would think that bnd could filter out this particular situation, however this is unlikely to happen.

**Comment by joeha480** [comment]
In some cases, the split-package warning is because the packageinfo files are included separately on the classpath. This appears to be a bug in either the bundle plugin or bnd, since the packageinfo file is included specifically for the purpose of building the package. It is possible to get around this by moving the packageinfo files from the resource sourceset to the java sourceset, however that's probably not worth the effort.

**Comment by joeha480** [comment]
I think I know what's going on now. It appears that the source sets, by default, include everything below the specified folder. So, assuming that all resources are in separate folders within the hierarchy (typically `resource-files`), bnd shouldn't have a problem with them if the source sets are configured properly:
```
sourceSets {
	main {
		java { srcDir 'src' exclude '**/resource-files/**/*'}
		resources { srcDir 'src' include '**/resource-files/**/*'}
	}
	test {
		java { srcDir 'test' exclude '**/resource-files/**/*'}
		resources { srcDir 'test' include '**/resource-files/**/*'} 
	}
	all {
		tasks[processResourcesTaskName].includeEmptyDirs = false
	}
}
```

**Comment by joeha480** [comment]
This will perhaps be fixed with #28

**Comment by joeha480** [comment]
Fixed in #28.
